README.md

```diff
@@ -49,16 +49,17 @@ This package is compatible with [Django User Agents](https://pypi.org/project/dj
 
 ## Settings
 
-`DLFE_APP_NAME` - used to define the application name that should be logged.
+`DLFA_LOG_PERSONALLY_IDENTIFIABLE_INFORMATION` - the formatter checks this setting to see if personally identifiable information should be logged. If this is not set to true, only the user's id is logged.
```

Comment:
Do we need the DLFA prefix for settings? Do we expect that we'll need to distinguish between a DFLA_APPLICATION and a BLAH_APPLICATION or similar?